query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/SourceInfoField.java

```diff
@@ -31,39 +30,32 @@
     private static final String FIELD_TYPE_NAME = "SourceInfo";
 
     private static final String DESCRIPTION =
-            "Contains various information such as if the source is reachable, and the source configuration";
+            "Contains various information such as if the source is reachable, and the source configuration.";
```

Comment:
Is inheritance even worth it at this point if they're all just sharing one field?